store/sqlstore/upgrade.go

```diff
@@ -516,9 +516,6 @@ func upgradeDatabaseToVersion49(sqlStore *SqlStore) {
 
 func upgradeDatabaseToVersion410(sqlStore *SqlStore) {
 	if shouldPerformUpgrade(sqlStore, Version490, Version4100) {
-
-		sqlStore.RemoveIndexIfExists("Name_2", "Channels")
```

Comment:
Why the channel index removal as part of this PR?

Reply:
No related to this PR, but also i couldn't find anywhere who's adding an index named `Name_2` anywhere